editoast/src/fixtures.rs

```diff
@@ -273,16 +272,10 @@ pub mod tests {
     #[fixture]
     pub async fn project(db_pool: Data<DbPool>) -> TestFixture<Project> {
         let project_model = Project {
-            name: Some("test_project".into()),
-            objectives: Some("".into()),
-            description: Some("".into()),
-            funders: Some("".into()),
-            budget: Some(0),
-            tags: Some(vec![]),
-            creation_date: Some(Utc::now().naive_utc()),
+            name: "test_project".into(),
```

Comment:
```suggestion
let project_cs = Project::default().into_changeset().name("test_project".to_owned());
TestFixture::create(project_cs, db_pool).await
```
Admittedly a `fill_defaults()` function could be a useful addition to changesets.

On the other hand, I feel like this is more a DB problem. If we're ok with creating a project just by providing its name, then all the other columns should have a default value defined in the SQL schema. In this case, I'd suggest creating a migration that adds these default values to the `project` schema. We could then write:


```rust
let project_cs = Project::changeset().name("test_project".to_owned());
TestFixture::create(project_cs, db_pool).await
```

That way we avoid deriving `Default` for `Project` which doesn't make sense because of its `id`.